docs/static/include/pluginbody.asciidoc

```diff
@@ -697,6 +697,57 @@ For more examples of output plugins, see the https://github.com/logstash-plugins
 
 endif::receive_method[]
 
+ifdef::event_factory_method[]
+
+===== `event_factory` Method
+
+Beginning with {ls} 7.14, we introduced an <<event-api-event-factory>> and deprecated direct access to `LogStash::Event.new`.
+A plugin's `EventFactory` can be accessed with the `event_factory` method, and can be used to create new events.
+
+===== `targeted_event_factory` Method
+
+Similarly, plugins that provide a `target` option can get a targeted `EventFactory` using the `targeted_event_factory` method.
+A targeted event factory will create events by setting the target field to the provided mapping instead of populating root-level fields.
+
+ifdef::event_factory_method[]
+
+===== `logger` Method
+
+Plugins have access to a logger, which can be used to emit structured logs.
+Plugins can emit logs at one of 5 different levels:
```

Comment:
```suggestion
Plugins can emit logs at one of five different levels:
```
Numbers one to nine are spelled out. (This is one of those grammar rules that I don't like, but I didn't make it.)  :-)